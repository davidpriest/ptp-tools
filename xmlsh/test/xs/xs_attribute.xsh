#
# Test of xs:integer
import commands xs=xs
a=xs:attribute(foo bar)
xtype $a
echo $a
echo <[ element { "elem" } { $a } ]>

b=xs:attribute( QName( foo http://uri bar ) spam )
echo <[ element { "elem" } { $b } ]>
