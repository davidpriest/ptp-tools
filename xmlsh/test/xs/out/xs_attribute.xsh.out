attribute()
bar
<elem foo="bar"/>
<elem xmlns:foo="http://uri" foo:bar="spam"/>
